docs/client.md

```diff
@@ -23,37 +23,39 @@ You can use the full commands (stake) or the short form (s) as shown below.
 
 Supported commands:
 
-    stake|s <amount> <address> <password>
-
 Stake some RAZORs
 
-    unstake|u <accountId>
+    stake --address <address> --amount <amount>
+    
+Delegate RAZORs
 
-Unstake all RAZORs
+    delegate --address <address> --amount <amount> --stakerId <staker_id>
 
-    withdraw|w <accountId>
+unstake some RAZORs
 
-Withdraw all RAZORs. Make sure RAZORs are unstaked and unlocked
+    unstake --address <address> --stakerId <staker_id> --amount <amount_of_sRZRs/RZRs> --autoWithdraw
 
-    vote|v <account> <password>
+withdraw RAZORs
 
-Start monitoring contract, commit, vote, propose and dispute automatically
+    withdraw --address <address> --stakerId <staker_id>
+    
+Delegate some RAZORs
 
-    transfer|t <to> <amount> <from> <password>
+    setDelegation --address <address> --status <true_or_false> --commission <commission>
 
-transfer RAZORs
+Activate Delegation
```

Comment:
Headings are still not correct. The heading of command should come first, then the explanation and then the command.
### Activate Delegation
The staker can choose to activate delegation if he/she likes. 
```
./razor setDelegation  --address <address> --status <true_or_false> --commission <commission_percentage>
```